src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+const CHECK_STATUS = 'check_status';
```

Comment:
- [ ] As it is a requirement for this project, " `Write your categories actions and reducer (in /src/redux/categories/categories.js) using the ducks pattern.` ", please kindly correct your actions to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Something like so: 
`CHECK_STATUS = bookstore/categories/CHECK_STATUS`

Reply:
All requested changes have been resolved @t-yanick